### DISCUSSION TITLE
What's the difference between op retries and run retries?

### DISCUSSION THREAD

**Discussion by sryza** [dagster-io/dagster#17594] [Q&A] [answered]
Does one limit the other?

**Comment by yuhan** [comment, marked as answer]
Op retries: https://docs.dagster.io/concepts/ops-jobs-graphs/op-retries
```python
@job(op_retry_policy=RetryPolicy(max_retries=1))
```
This will rerun in the same run, which is lighter weight.

---

Run retries: https://docs.dagster.io/deployment/run-retries
```python
@job(tags={"dagster/max_retries": 3, "dagster/retry_strategy": "FROM_FAILURE"})
```
Run retries will create a new run based on the retry strategy.
